torchao/prototype/grouped_mm/__init__.py

```diff
@@ -0,0 +1,370 @@
+from typing import Optional, Tuple
+
+import torch
+
+from torchao.float8.config import (
+    Float8LinearConfig,
+    Float8LinearRecipeName,
+    ScalingGranularity,
+)
+from torchao.float8.float8_utils import tensor_to_scale, to_fp8_saturated
+
+
+def _grouped_scaled_mm(
```

Comment:
nit: can we move this out of `__init__.py` into its own file?

Reply:
Done